package.json

```diff
@@ -17,7 +17,8 @@
     "prep-remote": "./scripts/prep-remote.sh",
     "type-check": "tsc --noEmit",
     "type-check:watch": "npm run type-check -- --watch",
-    "test": "jest --rootDir=. --config=config/jest.config.js"
+    "test": "jest --rootDir=. --config=config/jest.config.js",
+    "cypress:open": "cypress open"
```

Comment:
Like @eriknelson mentioned -> we would benefit from this script running headless instead.

Reply:
Updated scripts, so we have 2 commands to run unit or integration tests and one for running them all